Problem: upgrading aws-* crates to 0.14 fails

https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true
```
Compiling tmkms-nitro-helper v0.3.0 (/home/runner/work/tmkms-light/tmkms-light/providers/nitro/nitro-helper)
[429](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:430)
error[E0599]: no method named `provide_credentials` found for struct `ImdsCredentialsProvider` in the current scope
[430](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:431)
--> providers/nitro/nitro-helper/src/key_utils.rs:24:30
[431](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:432)
|
[432](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:433)
24|             .block_on(client.provide_credentials())
[433](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:434)
| ^^^^^^^^^^^^^^^^^^^method not found in `ImdsCredentialsProvider`
[434](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:435)
|
[435](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:436)
::: /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/aws-types-0.13.0/src/credentials/provider.rs:186:8
[436](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:437)
|
[437](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:438)
186|     fn provide_credentials<'a>(&'a self) -> future::ProvideCredentials<'a>
[438](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:439)
| -------------------the method is available for `ImdsCredentialsProvider` here
[439](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:440)
|
[440](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:441)
= help: items from traits can only be used if the trait is in scope
[441](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:442)
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
[442](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:443)
|
[443](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:444)
10| use aws_types::credentials::provider::ProvideCredentials;
[444](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:445)
|
[445](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:446)
[446](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:447)
error: unused import: `aws_types::credentials::ProvideCredentials`
[447](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:448)
--> providers/nitro/nitro-helper/src/key_utils.rs:12:9
[448](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:449)
|
[449](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:450)
12|     use aws_types::credentials::ProvideCredentials;
[450](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:451)
| ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[451](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:452)
|
[452](https://github.com/crypto-com/tmkms-light/runs/6869660401?check_suite_focus=true#step:5:453)
= note: `-D unused-imports` implied by `-D warnings`
```


likely the same cause as https://github.com/crypto-com/tmkms-light/issues/304 (to bump all crates together)

--- a/providers/nitro/nitro-helper/Cargo.toml
+++ b/providers/nitro/nitro-helper/Cargo.toml
@@ -5,8 +5,8 @@ authors = [ "Tomas Tauber <2410580+tomtau@users.noreply.github.com>" ]
 edition = "2021"
 
 [dependencies]
-aws-config = "0.12"
-aws-types = "0.12"
+aws-config = "0.15"
+aws-types = "0.15"
 ctrlc = "3"
 chrono = "0.4"
 ed25519-dalek = "1"

